oeps/oep-0001.rst

```diff
@@ -85,25 +85,25 @@ Authors
 
 Each OEP must have at least one Author: someone who writes the OEP using the
 style and format described here, shepherds the discussions in the appropriate
-forums, and attempts to build community consensus around the idea.  
+forums, and attempts to build community consensus around the idea.
```

Comment:
Were these changes due to warnings, or just because of your editor?

Reply:
Trailing space fixes were due to my editor.